src/System.Numerics.Vectors/src/System.Numerics.Vectors.csproj

```diff
@@ -19,6 +19,15 @@
     <PackageDestination Include="lib/portable-net45+win8+wp8+wpa81">
       <TargetFramework>portable-net45+win8+wp8+wpa81</TargetFramework>
     </PackageDestination>
+    <!-- Use the documentation file generated by this project for the reference assembly. -->
+    <FilesToPackage Include="$(DocumentationFile)">
+      <TargetPath>ref/netstandard1.0</TargetPath>
```

Comment:
Add a comment about how this needs to be kept in sync with all the PackageTargetFrameworks used by the reference assembly projects.